src/vma/dev/qp_mgr.cpp

```diff
@@ -587,8 +592,13 @@ void qp_mgr::mlx5_send(vma_ibv_send_wr *p_send_wqe)
 	 * implementations may use move-string-buffer assembler instructions,
 	 * which do not guarantee order of copying.
 	 */
+	if (likely(!m_b_is_hypervisor)) {
```

Comment:
The issue is not related to the sse3 itself sse3 works fine. The problem is accessing to the bf using sse in HV.